deps/rabbit/src/rabbit_core_ff.erl

```diff
@@ -132,13 +132,10 @@
       Ret :: rabbit_feature_flags:enable_callback_ret().
 direct_exchange_routing_v2_enable(#{feature_name := FeatureName}) ->
     TableName = rabbit_index_route,
-    ok = rabbit_table:wait([rabbit_route, rabbit_exchange], _Retry = true),
```

Comment:
The wait for `rabbit_exchange` table seems to have been lost in a rebase. It was added very recently by @ansd in #6847.